### PR TITLE
Add OtlpExporter as an option to ASP.NET example application

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Add OtlpExporter as an option to ASP.NET example application
- Updated bindingRedirect for System.Diagnostics.DiagnosticSource to 5.0.0.1. We recently updated Common.props to use DiagnosticSource Pacakge version of 5.0.1. The .NET framework of this package has updated the assembly version to 5.0.0.1

### CODE REVIEW

File: examples/AspNet/Global.asax.cs [hunk 1]
```diff
@@ -53,6 +53,17 @@ protected void Application_Start()
                         zipkinOptions.Endpoint = new Uri(ConfigurationManager.AppSettings["ZipkinEndpoint"]);
                     });
                     break;
+                case "otlp":
+                    // Adding the OtlpExporter creates a GrpcChannel.
+                    // This switch must be set before creating a GrpcChannel/HttpClient when calling an insecure gRPC service.
+                    // See: https://docs.microsoft.com/aspnet/core/grpc/troubleshoot#call-insecure-grpc-services-with-net-core-client
+                    AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
```
Comment (cijothomas) [inline examples/AspNet/Global.asax.cs:60]: Is this relevant for asp.net?

Comment (utpilla) [reply examples/AspNet/Global.asax.cs:60]: Yeah I believe this is to ensure that we have support for unencrypted gRPC channels.  We set this AppContext switch right before adding an OtlpExporter in other parts of the solution too.

Comment (utpilla) [reply examples/AspNet/Global.asax.cs]: You're right! This is only needed for gRPC net client. I have removed it.

File: examples/AspNet/Web.config [hunk 2]
```diff
@@ -49,7 +50,7 @@
 			</dependentAssembly>
 			<dependentAssembly>
 				<assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="CC7B13FFCD2DDD51" culture="neutral"/>
-				<bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.0"/>
+				<bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.1"/>
```
Comment (cijothomas) [inline examples/AspNet/Web.config:53]: Lets update PR description to indicate why this was needed

**Review by cijothomas — APPROVED** [review]

File: examples/AspNet/Web.config (hunk 2 above)
Comment (utpilla) [reply examples/AspNet/Web.config:53]: Updated the description.

**Pull request merged**